common/rdm/RDMAPI.cpp

```diff
@@ -2556,9 +2556,131 @@ bool RDMAPI::SetDnsDomainName(
     error);
 }
 
+/*
+ * @brief Fetch the dimmer curve
+ * @param uid the UID to fetch the dimmer curve information for
+ * @param sub_device the sub device to use
+ * @param callback the callback to invoke when this request completes
+ * @param error a pointer to a string which it set if an error occurs
```

Comment:
I don't know if this typo exists throughout, but it should be "which IS set", can you fix it wherever necessary please.

Reply:
Looks like it was present across all the functions. Changed all occurrences in RDMAPI.cpp